clang/docs/LanguageExtensions.rst

```diff
@@ -664,6 +664,7 @@ Unless specified otherwise operation(±0) = ±0 and operation(±infinity) = ±in
  T __builtin_elementwise_cosh(T x)           return the hyperbolic cosine of angle x in radians               floating point types
  T __builtin_elementwise_tanh(T x)           return the hyperbolic tangent of angle x in radians              floating point types
  T __builtin_elementwise_floor(T x)          return the largest integral value less than or equal to x        floating point types
+ T __builtin_elementwise_length(T x)         return the length of the specified floating-point vector         floating point types
```

Comment:
This doesn't match `__builtin_hlsl_elementwise_length`  defined in `clang/include/clang/Basic/Builtins.td` Also we have not typically documented hlsl builtins as language extentions because this is intended for clang c dialect.